docs/canonicalk8s/snap/reference/crypto.md

```diff
@@ -0,0 +1,87 @@
+# Cryptography
+
+This reference page provides an overview of the {{product}} cryptography
```

Comment:
I think this page is explanation. Reference would not include the level of explanation of you have included here. I think its great info just need to move it under /snap/explanation/security :)

Reply:
I would also expand on this intro a bit more